README.md

```diff
@@ -41,6 +41,46 @@ The Image Tool provides three functions within the main script:
 
 Use the [Quick Start](site/quickstart.md) guide to create a Linux based WebLogic Docker image.
 
+## Building From Source
+
+The Image Tool installer is available for download on the [Releases](releases) page.  If you want
+to build the installer from source instead of downloading it, follow these instructions:
+- Download and install JDK 8u261+
+- Download and install Maven 3.6.3+
+- Clone this repository to your local environment using one of the options under `Code` near the top of this page.
+- From inside the top-level directory of the cloned project, `weblogic-image-tool`, using Maven, execute one or 
+more of these phases:
+    - `validate` - Validate the project is correct and all necessary information is available.
+    - `compile`  - Compile the source code.
+    - `test`     - Test the compiled source code using the JUnit5 framework.
+    - `package`  - Create the installer ZIP file, `imagetool.zip`.
+    - `verify`   - Run integration tests using the JUnit5 framework.
+    - `clean`    - Restore the source by removing any items created by `package` or another phase of the build.
+    
+**Note:** Maven executes build phases sequentially, `validate`, `compile`, `test`, `package`, `verify`, such that 
+running `verify` will run all of these phases from `validate` through `package` before executing `verify`.
+
+Because the `package` phase comes before the `verify` phase, it is not necessary to run the integration tests to create 
+the Image Tool installer.  If you are making changes and want to validate those changes in your environment, you will 
+need to do some additional setup before running the `verify` phase because several of the integration tests require 
+access to the Oracle Technology Network.  To run the integration tests in the 
+`verify` phase, you must specify three environment variables, `ORACLE_SUPPORT_USERNAME`, `ORACLE_SUPPORT_PASSWORD`, 
+and `STAGING_DIR`.  The first two, Oracle Support user name and password, are used to connect to Oracle OTN for patches.
+The third, `STAGING_DIR`, should be a local folder where WebLogic Server installers, JDK installers, and pre-downloaded 
+patches can be found.  The files required in the `STAGING_DIR` depend on which tests that you want to run.  
```

Comment:
Why not CACHE_DIR

Reply:
Cache directory is something else.  Some of the files in the staging directory will be moved into the cache directory by some of the IT tests.

Reply:
Where are these env vars ORACLE_SUPPORT_USERNAME, ORACLE_SUPPORT_PASSWORD, and STAGING_DIR used?

Reply:
@jshum2479 I don't understand your question.  This sentence tells how they are used and that they are used only in the integration tests.  So, I do not understand what you mean.

Reply:
I searched the code and want to find how the environment variable is being used and cannot find it.

Reply:
Lines 248 and 249 of ITImagetool.java.

Reply:
Never mind, I was trying to see where the code is using these variables and I was in the wrong place.  Now, I found how they are used